### PR TITLE
Adding ELK-STACK Tempalte

### PR DESCRIPTION
**Pull request by Zahaby-magalix**

### CODE REVIEW

**Comment by Zahaby-magalix** [comment]
ELK-Twitter template 
and removing get-started category (requested by @omar)

File: stable/elasticsearch/template.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+				"value": "lAYjJj4bRoi4O70ZR9lwyw17O"
+			}, {
+				"name": "TWITTER_CONSUMER_SECRET",
+				"value": "SmFIOjw2a7v0hI9uQkOAhzNfcdmN0rSuShcLcqe7CZ2FHxQEJS"
+			}, {
+				"name": "TWITTER_OAUTH_TOKEN",
+				"value": "205062031-fZ1jDZYr4V20d4F9QoNjnHihvuVpMNnLitPoDrkM"
+			}, {
+				"name": "TWITTER_OAUTH_TOKEN_SECRET",
+				"value": "8a8hbT1stm55NNSbNqkus38698cK7mYQ73cr1SuyxFV1G"
```
Comment (gagliardetto) [inline stable/elasticsearch/template.json]: Are these real keys?

**Comment by Zahaby-magalix** [comment]
@gagliardetto 
Please check the updated file

**Comment by Zahaby-magalix** [comment]
@gagliardetto 

in the README: All of the copied parts from other articles, has been removed, and rephrased. 
Please take a look . 

Thanks.

**Pull request merged**